Week3/homework/6-step3.js

```diff
@@ -1,6 +1,18 @@
-'use strict';
-
+"use strict";
+const arr1d = [25, 32, 35];
 const arr2d = [[1, 2], [3, 4], [5, 6]];
-const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
+const multiDimensionalArr = [[1, 2, [200, 300]], [3, 4], [5, [500, [600]], 6]];
+
+function printArr(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    if (Array.isArray(arr[i])) {
+      printArr(arr[i]);
+    } else {
+      console.log(arr[i]);
+    }
+  }
+}
```

Comment:
👍 Excellent!  Nice use of recursion to get the job done.  
And I love that you created your own more complicated arr to really give your function a good test.  
Really excellent work.  You are the only one to have done this. 
In my work, when you have to figure this stuff out yourself, coming up with good test data is a creative challenge, and being hard on your code very important.
👏